### ISSUE TITLE
arm64 Docker support

### ISSUE THREAD

**Issue by probablypablito** [jjjchens235/bing-rewards#336]
Currently the Docker image only supports amd64. 

I attempted compiling the image using `sudo docker build .` but I get an exec error for selenium.

**Comment by Killerherts** [comment]
I'll look into it for you

**Comment by SebastianGode** [comment]
Changing the Dockerfile Run part to the following will fix the issue. However it clearly is a dirty fix. The Chromium package seems to have a different path on arm64 which raises the selenium error.

```
RUN set -ex \
    && apt-get update --no-install-recommends -y \
    && apt-get install --no-install-recommends -y  \
    chromium-driver \
    chromium \
    git \
    vim \
    nano \
    tzdata \
    cron \
    && mkdir -p /usr/lib/chromium-browser/ \
    && ln -s /usr/bin/chromedriver /usr/lib/chromium-browser/chromedriver \
```

**Comment by Killerherts** [comment]
Yeah there is going to be a bit more work to get this going than I anticipated. I do have a buildx test of the current container at killerhert/armtest

**Comment by dionperera** [comment]
> Changing the Dockerfile Run part to the following will fix the issue. However it clearly is a dirty fix. The Chromium package seems to have a different path on arm64 which raises the selenium error.
> 
> ```
> RUN set -ex \
>     && apt-get update --no-install-recommends -y \
>     && apt-get install --no-install-recommends -y  \
>     chromium-driver \
>     chromium \
>     git \
>     vim \
>     nano \
>     tzdata \
>     cron \
>     && mkdir -p /usr/lib/chromium-browser/ \
>     && ln -s /usr/bin/chromedriver /usr/lib/chromium-browser/chromedriver \
> ...
> ```

This works for me!

**Comment by Killerherts** [comment]
I'll have to check it to see if it has the permission issue I was running into. On arm it seems my user is not able to copy in the cron doc. 

@dionperera  is cron working for you?

**Comment by dionperera** [comment]
> @dionperera is cron working for you?

Yes, I manually ran it last night, but it automatically started today.

**Comment by SebastianGode** [comment]
Same for me, needed to run it manually after the first day but now worked two days in a row.
